pytest.ini.example

```diff
@@ -15,9 +15,12 @@ env =
     azext_iot_testrg=
     azext_iot_testhub=
     azext_iot_testdps=
-    azext_iot_teststorageuri=
-    azext_iot_teststorageid=
+    azext_iot_identity_teststorageid=
```

Comment:
azext_iot_identity_teststorageid is a synonym for azext_iot_identity_teststoragaccount?

Reply:
I mean would be good to be consistent with naming